frontend/src/components/admin/testManagementConfigMenu/TestManagementConfigMenu.js

```diff
@@ -0,0 +1,416 @@
+                      <ListItem>
+                        <FormattedMessage id="configuration.method.rename.explain" />
+                      </ListItem>
+                    </UnorderedList>
+                  </ListItem>
+                </ClickableTile>
+              </UnorderedList>
+            </Column>
+          </Grid>
+          <br />
+          <hr />
+          <br />
+          <Grid fullWidth={true}>
+            <Column lg={16} md={8} sm={4}>
+              <Section>
+                <Section>
+                  <Section>
+                    <Heading>
+                      <FormattedMessage id="configuration.test.management.organization" />
+                    </Heading>
+                  </Section>
+                </Section>
+              </Section>
+            </Column>
+          </Grid>
+          <br />
+          <hr />
+          <br />
+          <Grid fullWidth={true}>
+            <Column lg={16} md={8} sm={4}>
+              <UnorderedList>
+                <ClickableTile>
+                  <ListItem
+                    onClick={() => {
+                      window.location.assign("/admin#TestCatalog");
+                    }}
+                  >
+                    <FormattedMessage id="configuration.test.catalog" />
+                    <UnorderedList nested>
+                      <ListItem>
+                        <FormattedMessage id="configuration.test.catalog.explain" />
+                      </ListItem>
+                    </UnorderedList>
+                  </ListItem>
+                </ClickableTile>
+                <br />
+                <ClickableTile>
+                  <ListItem
+                    onClick={() => {
+                      window.location.assign("/admin#MethodManagement");
+                    }}
+                  >
+                    <FormattedMessage id="configuration.method" />
+                    <UnorderedList nested>
+                      <ListItem>
+                        <FormattedMessage id="configuration.method.explain" />
+                      </ListItem>
+                    </UnorderedList>
+                  </ListItem>
+                </ClickableTile>
+                <br />
+                <ClickableTile>
+                  <ListItem
+                  // onClick={() => {
+                  //   window.location.assign("/admin#TestAdd");
+                  // }}
+                  >
+                    <FormattedMessage id="configuration.test.add" />
+                    <UnorderedList nested>
+                      <ListItem>
+                        <FormattedMessage id="configuration.test.add.explain" />
+                      </ListItem>
+                    </UnorderedList>
+                  </ListItem>
+                </ClickableTile>
+                <br />
+                <ClickableTile>
+                  <ListItem
+                  // onClick={() => {
+                  //   window.location.assign("/admin#TestModifyEntry");
```

Comment:
in the case when the component doesnt exist or not functional in React , just link to the old JSP page ie `api/OpenELIS-Global/TestModifyEntry`